lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -67,26 +61,8 @@
     assertThat(LocationServices.SettingsApi).isNull();
   }
 
-  @Test public void disconnect_shouldUnregisterLocationUpdateListeners() throws Exception {
-    client.connect();
-    LocationServices.FusedLocationApi.requestLocationUpdates(LocationRequest.create(),
-        new LocationListener() {
-          @Override public void onLocationChanged(Location location) {
-          }
-
-          @Override public void onProviderDisabled(String provider) {
-          }
-
-          @Override public void onProviderEnabled(String provider) {
-          }
-        });
-
-    client.disconnect();
-    LocationManager lm = (LocationManager) application.getSystemService(LOCATION_SERVICE);
-    assertThat(shadowOf(lm).getRequestLocationUpdateListeners()).isEmpty();
-  }
-
-  @Test public void isConnected_shouldReturnFalseBeforeConnected() throws Exception {
+  @Test @Ignore("Failing after removal of disconnect_shouldUnregisterLocationUpdateListeners")
```

Comment:
This test is not failing for me.

It seems like it should be fine since the `tearDown()` method is calling `client.disconnect()` after each test method.